Sources/Pioneer/GraphQL/Extensions/Field+Middleware.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  Field+Middleware.swift
+//  pioneer
+//
+//  Created by d-exclaimation on 00:13.
+//
+
+import class Graphiti.ArgumentComponent
+import struct Graphiti.ArgumentComponentBuilder
+import typealias Graphiti.ConcurrentResolve
+import class Graphiti.Field
+import typealias Graphiti.SyncResolve
+
+public extension Field where FieldType: Encodable {
```

Comment:
There's no comment explain any of these lines of use, maybe add some to explain the middleware ordering or extract that into its own function possible